calculator.js

```diff
@@ -0,0 +1,10 @@
+const add = (a, b) => a + b;
+
+const substract = (a, b) => a - b;
+
+const multiply = (a, b) => a * b;
+
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/bakersen/code-quality-badges/pullRequest?prid=8111319)
```suggestion
  return a * b;
```